tests/jballerina-unit-test/src/test/resources/test-src/klass/simple_class.bal

```diff
@@ -70,6 +70,33 @@ class Per {
     int i = 0;
 }
 
+final int classI = 111222;
+
+class ModuleVariableReferencingClass {
+    int i = classI;
+}
+
+function value(int k = classI) returns int {
+    return k;
+}
+
+ModuleVariableReferencingClass c1 = new;
+
+function testModuleVariableReferencingClass() {
+    ModuleVariableReferencingClass c = new;
+    assertEquality(c.i, 111222);
+    assertEquality(c1.i, 111222);
+}
+
+function testLocalObjectConstructorReferencingModuleVariable() {
```

Comment:
Can we make the names uniform? They are both doing the same thing, just different constructs, right?

Reply:
Updated as `testClassReferencingModuleVariable` and `testLocalObjectConstructorReferencingModuleVariable`